test-network-function/networking/suite.go

```diff
@@ -108,6 +108,7 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 		})
 
 		ginkgo.ReportAfterEach(results.RecordResult)
+		ginkgo.AfterEach(env.CloseLocalShellContext)
```

Comment:
why not call this once, if common/suite.go ?

Reply:
In case some command hangs, we don't want to impact all other test cases by using the same shell.